src/dataLoaders/components/BucketsDropdown.tsx

```diff
@@ -53,7 +53,7 @@ class BucketsDropdown extends PureComponent<Props> {
       return 'Choose a bucket'
     }
 
-    return buckets.find(bucket => bucket.id === selectedBucketID).name
+    return buckets.find(bucket => bucket.id === selectedBucketID)?.name ?? ''
```

Comment:
`Array.find` can return `undefined`, therefore it is common sense to guard against it with empty string.